.github/workflows/operator-ci.yml

```diff
@@ -41,9 +41,12 @@ jobs:
           kubernetes version: v1.22.3
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
+      - name: Test operator with minikube
```

Comment:
I'd unify the naming of the two test phases. Something like "Test operator running locally" and "Test operator running in cluster" maybe?

Reply:
done